include/seastar/http/request.hh

```diff
@@ -152,4 +152,8 @@ struct request {
 
 } // namespace httpd
 
+namespace httpd {
+using request [[deprecated("Use http::request instead")]] = http::request;
```

Comment:
But you should be aware that virtually every user who ever used Seastar's http server would immediately suffer this deprecation.
Arguably, it makes sense that in an application that only uses Seastar' httpd (http server), it might be enough to use the httpd namespace and the application doesn't need to know that some of the classes are actually shared with the http client.
But you're right that in the long run, it's better that we don't have aliases - it will only add confusion.
Let's open the floor to other people's opinion on this.

Reply:
A deprecation notice is a small price to pay. We go to great length to avoid breaking the API without notice, so I think this is okay.